### DISCUSSION TITLE
YOURLS text misspelled

### DISCUSSION THREAD

**Discussion by janasco** [YOURLS/YOURLS#3408] [General]
Just notice that the phrase Y**0**URLS is misspelled in this line: https://github.com/YOURLS/YOURLS/blob/6b96c827e0e0c952cec60f996514d84fe3ffcfa8/includes/functions-upgrade.php#L8

I guess that should be **YOURLS**.

My little contribution. 😅

**Comment by LeoColomb** [comment]
@janasco Nice catch!
Would you like to open a pull request to correct that?
Thx!

**Comment by f1rf1n** [comment]
Pull request https://github.com/YOURLS/YOURLS/pull/3535
